CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.8.12)
 project(rapidcheck)
 enable_testing()
 
-option(RC_ENABLE_TESTS "Build RapidCheck tests" OFF)
+option(RC_ENABLE_TESTS "Build RapidCheck tests" ON)
```

Comment:
Most people are (I hope) RapidCheck users and not RapidCheck developers and will likely not want to build the tests by default.